custom_components/ha_hatch/manifest.json

```diff
@@ -5,7 +5,7 @@
   "issue_tracker": "https://github.com/dahlb/ha_hatch/issues",
   "codeowners": ["@dahlb"],
   "requirements": [],
-  "version": "1.11.1",
+  "version": "1.11.2",
```

Comment:
not sure why this and the next config_turn_on_media are listed in the pr when they match what's in master, might want to try rebase instead of merging master next time but it shouldn't mess anything up except annotations

Reply:
Yeah I don't know why they got pulled into this PR either.